doc/changelog/10-standard-library/13895-rename_int63.rst

```diff
@@ -0,0 +1,20 @@
+- **Deprecated:**
+
+  The naming convention for unsigned primitive integers is now ``uint63``
+  instead of ``int63``.
+
+  Thus, the ``Int63.v`` file was deprecated and replaced by ``Uint63.v``,
```

Comment:
```suggestion
  The ``Int63`` module is replaced by ``Uint63``,
```

The title of the item already says "deprecated".  Users may think more in terms of modules rather than files--you think?

Reply:
Sure.